spec/assets/workplace_credential_vc.md

```diff
@@ -0,0 +1,24 @@
+REQUIRED claim is only `displayName`. All other claims are OPTIONAL and might be omitted.
+
+```json
+{
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1",
+  ],
+  "type": [
+    "VerifiableCredential",
+    "WorkplaceCredential"
+  ],
+  "credentialSubject": {
+    // keys to match those defined in @context as agreed upon by interop partners
```

Comment:
I don't think user claims are dependent on `@context` at this point.
suggest paraphrasing to mean something like "user claims are defined in the schema agreed by the interop partners"